README.md

```
@@ -1,3 +1,70 @@
 # java-janggi
 
 장기 미션 저장소
+
+# 입출력 요구사항
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)